api/index.php

```diff
@@ -178,7 +178,7 @@ function api_send_pagination_header($query_params, $total_rows, $per_page, $page
 
     // If the URI doesn't include the url param (because the Apache config
     // hasn't been updated) we need to include it here to make valid links.
-    $link_base = $_SERVER["SCRIPT_URI"] . "?";
+    $link_base = $_SERVER["SCRIPT_NAME"] . "?";
```

Comment:
This will break navigation links because `$_SERVER["SCRIPT_NAME"]` is not a valid base URL:
```php
<?php
echo "<pre>";
echo "SCRIPT_URI: " . $_SERVER["SCRIPT_URI"] . "\n";
echo "SCRIPT_NAME: " . $_SERVER["SCRIPT_NAME"] . "\n";
```

Outputs:
```
SCRIPT_URI: https://www.pgdp.org/~cpeel/c/script_links.php
SCRIPT_NAME: /~cpeel/c/script_links.php
```

Reply:
OK. After some staring at the `$_SERVER` docs, I've opted to build `SCRIPT_URI` from more widely available pieces instead.